tests/TestCase/Datasource/SimplePaginatorTest.php

```diff
@@ -108,19 +108,19 @@ public function testPaginateCustomFindFieldsArray(): void
         ];
         $results = $this->Paginator->paginate($table, [], $settings);
 
-        $result = $results->toArray();
+        $result = $results->items()->toArray();
```

Comment:
Should the new object have a `toArray()` method on it?

Reply:
I can add it.